### ISSUE TITLE
Build Won't Play on SteamVR - Black Screen

### ISSUE THREAD

**Issue by dumbgamedev** [ValveSoftware/steamvr_unity_plugin#742]
**Setup:**
Unity: 2019.3.14f1
SteamVR: 1.11.13
Steam Unity Plugin Package: Latest version
Device: Vive OG
OS: Windows 10, PC

**Tried with:**
Single Pass/ Multipass
.Net2 / .Net4
Gamma/Linear Color Space
Mirror Mode/ No Mirror Mode
Computer, Steam, Unity restarts
Fresh Project
Older version of Unity: 2019.3.11f1

**Issue:**
Scene will load when hitting play within Unity. No issues there.

However, after creating a build and running the build, I always get a black screen and SteamVR does not launch. If I pre-launch SteamVR, it stays in the home area.

No unity logo is shown, straight to black.

There are no errors on building. Editor log attached.

Unfortunately, I am out of ideas.

[EditorLogs.txt](https://github.com/ValveSoftware/steamvr_unity_plugin/files/4689466/EditorLogs.txt)

**Comment by kisak-valve** [comment]
Hello @dumbgamedev, can you reproduce the issue with 2019.3.6f1, like https://github.com/ValveSoftware/steamvr_unity_plugin/issues/738#issuecomment-633811200?

**Comment by dumbgamedev** [comment]
Hi @kisak-valve , I will give it a try. I did read their comment first, so I tried mirror mode on/off, as well as a restart. I am going to upload my project to git, if someone from your end wants to give it a try. Its very small. In the meantime, I will download 2019.3.6f1. Although they said it worked for them on 2019.3.14f1, which I used.

Thanks for the quick reply! :)

**Comment by dumbgamedev** [comment]
Here is an example project in a zip. Its version 2019.3.14f1.

https://github.com/dumbgamedev/ExampleProject

**Comment by dumbgamedev** [comment]
I added looping audio to the scene, to if anything was happening despite display issues. The sound loop does play over the PC, so the scene must be loading.

**Comment by dumbgamedev** [comment]
Created a build with 2019.3.6f1. Same result.

Build here: https://github.com/dumbgamedev/ExampleProject

**Comment by jeffrey97** [comment]
Hi @dumbgamedev 
I ran into the exact same issue some days back with my 64-bit build on my Vive.

I checked your uploaded build and the issue is the same, as the 'OpenVRSettings.asset' file is missing in the 'StreamingAssets\SteamVR' folder in your build.

This is happening due to a missing code in SettingsCopier.cs file which is part of the Open VR XP plugin which is detailed in this link [https://github.com/ValveSoftware/steamvr_unity_plugin/issues/713#issuecomment-634241004](URL).

My issue was resolved after making this code change and rebuilding the Player. This issue would get fixed in the next release of the plugin.
